Assets/HoloToolkit-Examples/SpatialUnderstanding/Scripts/SpatialUnderstandingBasicCursor.cs

```diff
@@ -48,14 +48,14 @@ protected virtual void Awake()
 
         protected virtual void Start()
         {
-            gazeManager = GazeManager.Instance;
+            gaze = GazePointer.Instance;
```

Comment:
Not sure if it's worth caching the reference if we're just going to call it directly below